docs/pages/enroll-resources/application-access/cloud-apis/aws-console-without-agent.mdx

```diff
@@ -0,0 +1,379 @@
+---
+title: AWS Console and CLI access
+description: How to access AWS with Teleport.
+labels:
+ - how-to
+ - zero-trust
+---
+
+You can protect the AWS Management Console and AWS APIs with Teleport.
+This allows you to manage access to AWS infrastructure with Teleport features like Access Requests, Access Lists, and identity locking.
+You can also configure Teleport to provide different levels of AWS access automatically when users authenticate using your single sign-on solution.
+
+<Admonition type="tip">
+When compared with the [AWS Console and CLI access via Teleport Agent](aws-console.mdx) method,
+this guide does not require a Teleport Agent to be installed and provides 100% compatibility with AWS SDK based tools (e.g., terraform).
+However, it does not record the requests in Teleport's audit logs, only that a session was started by a given user.
+</Admonition>
+
+This guide will explain how to:
+
+- Configure AWS Console and CLI access with Teleport using Roles Anywhere
+- Access the AWS Management Console
+- Access the AWS Command Line Interface (CLI)
+- Access applications using AWS SDKs
+
+## How it works
+
+Teleport leverages AWS IAM Roles Anywhere to issue temporary credentials for assuming target IAM Roles.
+
+Access is managed through Teleport's RBAC policies, ensuring credentials are only generated for authorized users and roles.
+
+For web console access, you can navigate to the resources page in Teleport Web UI and click on the AWS Application which is named after the Roles Anywhere Profile.
+
+For CLI and SDK based access, you must use `tsh` to obtain the credentials
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- Permissions in AWS to:
+  - create IAM Roles Anywhere Trust Anchor
+  - create IAM Roles Anywhere Profiles
+  - create IAM Roles
+- `aws` command line interface (CLI) tool in PATH. Read the AWS documentation to
+  [install or update the latest version of the AWS
+  CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
+
+## Step 1/4. Configure AWS IAM Roles Anywhere integration
+
+In this section, you will configure AWS IAM resources to allow Teleport to issue AWS credentials.
+
+<Admonition type="note">
+Instead of doing this step manually, you can navigate to your Teleport Web UI,
+click on Enroll New Resource in Resources listing page,
+and follow the guide after clicking on the AWS CLI/Console Access tile.
+</Admonition>
```

Comment:
It seems like our docs guide is pushing people towards the manual setup and only mentioning the guided flow.

I would think we want it to be the other way around. Push people towards the flow we control, and link to manual setup instructions for power users who want to know exactly what's happening behind the scenes.